app/controllers/lesson_completions_controller.rb

```diff
@@ -15,7 +15,9 @@ def destroy
   private
 
   def lesson_completion
```

Comment:
I think this one looks a little disjointed, maybe this would be cleaner looking?
```
LessonCompletion.where(
  student_id: current_user.id,
  lesson_id: @lesson.id
).first
```